src/NGen.cpp

```diff
@@ -265,7 +265,12 @@ int main(int argc, char *argv[]) {
                              r_c->get_output_line_for_timestep(output_time_index)+"\n";
         r_c->write_output(output);
         //TODO put this somewhere else.  For now, just trying to ensure we get m^3/s into nexus output
-        response *= (catchment_collection->get_feature(id)->get_property("areasqkm").as_real_number() * 1000000);
+        try{
+          response *= (catchment_collection->get_feature(id)->get_property("areasqkm").as_real_number() * 1000000);
+        }catch(std::invalid_argument &e)
```

Comment:
While I'm ok with this, I am curious how much of an overhead the use of an exception here might introduce.  I've only done a quick investigation, but I did find [this](https://pspdfkit.com/blog/2020/performance-overhead-of-exceptions-in-cpp/), suggesting it could be fairly large (relatively speaking).  

Since this is in a loop of all catchments, within another loop of all time steps, it may be worth considering a runtime check of which key is present, though perhaps as a future improvement.

Reply:
This is really just to support our existing (legacy) example files.  I'm OK with reverting this to a single key once the example files get changed.  This has changed a few times in the past, and I figured for now this was the quickest way to support both.

And which key is present presents the challenge that you need to check the key for each catchment and use the right one in the right place.  It would probably be OK to assume it is used consistently within a file though...

Reply:
Ah, I see.  I do like the idea of a having a variable and then have it be updated in the exception case, assuming that the key is consistent throughout a file.  But the (amortized) overhead I was worried about sounds like it is very low, given the additional context, so I'll leave it up to you.